modules/network/aws_vpc/0.1/facets.yaml

```
@@ -28,12 +29,16 @@ spec:
       x-ui-immutable: true
       x-ui-overrides-only: true
       x-ui-fetch-az: true
+      items:
+        type: string
     vpc_cidr:
       title: VPC CIDR
       description: CIDR block for the vpc
       x-ui-placeholder: e.g. 10.45.0.0/16
-      pattern: ^(10.(\d{1,3}.){2}\d{1,3}/(2[0-8]|[89]|1[0-9])|172.(1[6-9]|2[0-9]|3[01]).(\d{1,3}.){1}\d{1,3}/(2[0-8]|[12][0-9])|192.168.(\d{1,3}.){1}\d{1,3}/(2[0-8]|[12][0-9]|1[6-9]))$
-      x-ui-error-message: The provided CIDR block is invalid. Please ensure it is a valid private IP range in CIDR notation
+      pattern: 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)